moveit_ros/planning/rdf_loader/test/test_rdf_integration.cpp

```diff
@@ -66,6 +66,30 @@ TEST(RDFIntegration, topic_based)
   EXPECT_EQ("gonzo", loader.getSRDF()->getName());
 }
 
+// RDFLoader should successfully load URDF and SRDF strings from a ROS topic when the node that is passed to it is
+// spinning.
+// GIVEN a node that has been added to an executor that is spinning on another thread
+// WHEN the RDFLoader is created
+// THEN the RDFLoader should return non-null values for the URDF and SRDF model.
```

Comment:
I'm sorry to keep asking for changes but can you put these comments in the body of the test.  The first two lines go at the beginning and the WHEN and THEN lines go right before the respective parts.